scaling_performance/scaling_cluster_metrics.adoc

```diff
@@ -0,0 +1,85 @@
+[[scaling-performance-cluster-metrics]]
+= Scaling Cluster Metrics
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+
+== Overview
+
+{product-title} exposes metrics that can be collected and stored in back-ends by
+link:https://github.com/GoogleCloudPlatform/heapster[Heapster]. As an
+{product-title} administrator, you can view containers and components metrics in
+one user interface. These metrics are also used by
+xref:../dev_guide/pod_autoscaling.adoc#dev-guide-pod-autoscaling[horizontal pod
+autoscalers] in order to determine when and how to scale.
+
+This topic provides information for scaling the metrics components.
+
+[[cluster-metrics-scaling-openshift-metrics-pods]]
+== Scaling {product-title} Metrics Pods
+
+One set of metrics pods (Cassandra/Hawkular/Heapster) is able to monitor at
+least 10,000 pods.
```

Comment:
@ekuric @mffiedler I think we should say something like

In OpenShift 3.5, the default value for METRIC_RESOLUTION was changed from 15 seconds to 30 seconds.  This allows for twice as many pods to be monitored, and thus we are raising the limit to 25,000 pods.

I can't find the BZ with the exact reasoning behind the 15->30s increase, but IIRC it was something like the web UI only shows 30s anyway, so 15s didn't actually add any value?